CONTRIBUTING.md

```diff
@@ -44,13 +53,8 @@ A project's directory should contain the following:
 A version directory should contain the following:
 
 1. Static files needed for the project to work.
-2. If there is no minified version of the main JS/CSS file, please create your own using this ([minification tool][3]).
-3. If there are official or expected source maps for the minified js, please include those in the folder.  Currently, the following projects officially support the `.map` files:
-  * angularjs
-  * jQuery
-  * mithril
-4. Do not upload useless files like demos, examples, licenses, readmes and any other files not being used in the production.
+2. To shrink JS/CSS, you can use YUI ([minification tool][3]).
```

Comment:
Seems very good @as-com!  Thought does drop top-line-comment, which is important for asset tracking & copyrights, very important since we contaminate files.
Maybe I should fork?